docs/app/Examples/Button/Labeled.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import {Button} from 'stardust';
+
+export default class extends Component {
+  render() {
+    return (
+      <Button className='labeled'>
+        <div className='ui button'>
+          <i className='heart icon'></i> Like
+        </div>
```

Comment:
Clarifying:

A Stardust `<Button />` renders a `div` with the classes `ui button`.  We should use the Stardust `<Button />` instead of the `<div className='ui button'>` in this example.